x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/column_data.tsx

```diff
@@ -4,79 +4,101 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import { ConfusionMatrix, PredictedClass } from '../../../../common/analytics';
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButtonEmpty, EuiButtonIcon, EuiPopover, EuiText } from '@elastic/eui';
+import { ConfusionMatrix } from '../../../../common/analytics';
 
 interface ColumnData {
   actual_class: string;
   actual_class_doc_count: number;
-  predicted_class?: string;
-  count?: number;
-  error_count?: number;
+  [key: string]: any;
 }
 
 export const ACTUAL_CLASS_ID = 'actual_class';
+export const OTHER_CLASS_ID = 'other';
```

Comment:
We should probably internationalize this as well if possible, as it is used as the column label.

Reply:
Currently, these values are also used as keys in the objects used to populate the data grid so we can't internationalize them. Happy to look into it in a follow-up if that sounds alright. 👌